src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -376,7 +399,10 @@ function generateFrontendNavigationPerformanceEventView(
   }
   savedQuery.query = stringifyQueryObject(conditions);
 
-  return EventView.fromNewQueryWithLocation(savedQuery, location);
+  const eventView = EventView.fromNewQueryWithLocation(savedQuery, location);
+  eventView.additionalConditions.addTagValues('event.type', ['transaction']);
+  eventView.additionalConditions.addTagValues('!transaction.op', ['pageload']);
```

Comment:
The `addTagValues()` methods is chainable so you  could do `.addTagValues().addTagValues()`